bin/tessel-2.js

```diff
@@ -203,8 +203,13 @@ makeCommand('run')
     default: false,
     help: 'Deploy a project containing all files within, including those not used by the program, excluding any files matched by non-negated rules in .tesselignore and including any files matched by rules in .tesselinclude. Program is started from specified file.'
   })
+  .option('rustCC', {
```

Comment:
I'll go with `rustcc` because it's shortest and, let's be honest, most users are very unlikely to go through the trouble of setting up their own cross compilers. It's pretty much just a developer flag.